app/app/settings.py

```diff
@@ -99,7 +99,23 @@
     'external_bounties',
     'dataviz',
     'impersonate',
-    'bounty_requests'
+    'bounty_requests',
+
+    # wagtail
+    'taggit',
```

Comment:
Should we include `TAGGIT_CASE_INSENSITIVE = env.bool('TAGGIT_CASE_INSENSITIVE', default=True)` somewhere in the settings?

Reply:
That's a good idea